### PR TITLE
refactor(title): FRMW-209 Clean Components of title CSS Class

### PR DESCRIPTION
**Pull request by ssalinas24**
JIRA: https://jira.rax.io/browse/FRMW-209

All required LGTMs can be found at #1329

### CODE REVIEW

File: src/rxApp/common.less [hunk 1]
```diff
@@ -233,6 +234,23 @@ h6 {
...
+    &    { font-size: 18px; } // h3
+    &.sm { font-size: 16px; } // h4
+    &.xs { font-size: 13px; } // h5
+
+    // Colors
+    &.subdued {
+        color: @subduedTitle;
+    }
+}//.title
+
```
Comment (CITguy) [inline src/rxApp/common.less]: Please revert this change. It is introducing duplicate style definitions (210-235 is same).

**Comment by bry4n** [comment]
LGTM

**Comment by CITguy** [comment]
LGTM, @ssalinas24 you good with visual changes in rackerlabs/encore-ui-screenshots#87 ?

**Comment by ssalinas24** [comment]
@CITguy Yes the Visual Changes are good. I confirmed with Eric and the line height on the headers are exactly what was intended.
